modules/end-user-guide/pages/benefits-of-pull-requests-review-in-che.adoc

```diff
@@ -12,7 +12,6 @@
 
 .Prerequisites
 * You have access to the repository hosted by your Git provider.
-* You use a {prod}-supported browser: Google Chrome or Mozilla Firefox.
```

Comment:
@max-cx is this line something we want to remove entirely? I was torn between removing it, or replacing it with something like

> You have access to a web browser.